x/stakeibc/keeper/host_zone.go

```diff
@@ -313,6 +318,42 @@ func (k Keeper) RemoveValidatorFromHostZone(ctx sdk.Context, chainId string, val
 	return errorsmod.Wrapf(types.ErrValidatorNotFound, errMsg)
 }
 
+// Checks if any validator's portion of the weight is greater than the cap
+func (k Keeper) CheckValidatorWeightsBelowCap(ctx sdk.Context, validators []*types.Validator) error {
+	// If there's only a few validators, don't enforce this yet
+	if len(validators) < MinValidatorsBeforeWeightCapCheck {
+		return nil
+	}
+
+	params := k.GetParams(ctx)
+	validatorWeightCap := float64(params.ValidatorWeightCap)
+
```

Comment:
Agreed on putting it in the param validation! imo it's reasonable to exclude, I don't think we'll run into this issue, and it might be slightly better to allow governance to make this param arbitrary